assets/src/edit-story/components/canvas/page.js

```diff
@@ -72,16 +85,67 @@ function Page() {
 			{ hasSelection && (
 				<Selection { ...selectionProps } />
 			) }
+			{ displayMoveable && (
+				<Moveable
+					target={ document.querySelector( '.selection' ) }
```

Comment:
Correct, this assumes a separate selection frame, the idea being simplifying the logic for now to avoid worrying about updating the original element and its "insides" while an event is happening.

Do you mean using `ref` for the target instead of `document.querySelector`? Tried using refs initially but this didn't seem to work right away so moved on to avoid debugging too much. Will circle back to it a bit later though. I'll look a bit more into the Moveable documentation as well.

Reply:
Agree. Let's circle back with ref->state.

Reply:
That's replaced now, using ref insteaf of `document.querySelector`.